templates/release.mako

```diff
@@ -85,5 +85,7 @@ Alternatively, if there is a package in your software repository then you should
 
 [hidden=Other Platforms, Source Code]
 [color=green][size=12pt]Source Code Export[/size][/color]
-[url=${groups["Win32"].mainFile.tarball}]Source Code[/url]
+[url=${sources["Unix"].url}]Source Code (Unix line endings)[/url]
+
+[url=${sources["Win"].url}]Source Code (Windows line endings)[/url]
```

Comment:
I assumed that Appveyor uses `autocrlf` to automatically clone the repository with Windows line endings but I'm not sure if that's actually the case since I found multiple instances where it says that Appveyor uses `input` as the default `autocrlf` which would mean that the source code is checked out with Unix line endings.

We can check that once the next RC is created.